view/frontend/web/js/customer.js

```diff
@@ -0,0 +1,23 @@
+/***
+ * Copyright © MagePal LLC. All rights reserved.
+ * See COPYING.txt for license details.
+ * https://www.magepal.com | support@magepal.com
+ */
+
+define([
+    'underscore',
+    'Magento_Customer/js/customer-data'
+], function (_, customerData) {
+    'use strict';
+    
+    window._learnq = window._learnq || [];
```

Comment:
do we need the `.window` here or just use `var` ?

Reply:
In a function, the variable will be a local variable to that function so you need "window." to get access to the global namespace of `_learnq`